packages/in_app_purchase/in_app_purchase_android/lib/src/in_app_purchase_android_platform_addition.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:in_app_purchase_platform_interface/in_app_purchase_platform_interface.dart';
+
+import '../billing_client_wrappers.dart';
+
+/// Contains InApp Purchase features that are only available on PlayStore.
+class InAppPurchaseAndroidPlatformAddition
+    extends InAppPurchasePlatformAddition {
+  /// Creates a [InAppPurchaseAndroidPlatformAddition] which uses the supplied
+  /// `BillingClient` to provide Android specific features.
+  InAppPurchaseAndroidPlatformAddition(this._billingClient) {
+    if (InAppPurchaseAndroidPlatformAddition.enablePendingPurchase) {
+      _billingClient.enablePendingPurchases();
+    }
+  }
+
+  /// Whether pending purchase is enabled.
+  ///
+  /// See also [enablePendingPurchases] for more on pending purchases.
+  static bool get enablePendingPurchase => _enablePendingPurchase;
```

Comment:
Since users will be getting the `InAppPurchaseAndroidPlatformAddition` from the provider, which is going to be a single instance, we can make `_enablePendingPurchase` an instance variable. And `enablePendingPurchases` below an instance method.

Reply:
As per offline discussion we decided to leave it static as it is important the `enablePendingPurchases()` method is called before initialising the `InAppPurchaseAndroidPlatform` class.